### PR TITLE
Add charge holy weapon

### PR DESCRIPTION
**Pull request by jayRyan24**
Will find out the data for crossing and submit it. 

REQUIRED SETTINGS: (I WILL UPDATE DOCS ONCE MERGED!)
```
holy_weapon:
  weapon_name: serrated khopesh
  icon_name: shark icon
  icon_container: pack
  icon_room: 2866
```

### CODE REVIEW

**Comment by jayRyan24** [comment]
My intention is to update this for Icons once I can get my hands on one!

**Review by rpherbig — CHANGES_REQUESTED** [review]

**Comment by rpherbig** [comment]
Looks good ✔️  - only the change to `custom_require` is required.

**Comment by jayRyan24** [comment]
> Looks good ✔️ - only the change to `custom_require` is required.

All good suggestions, the timer one was one I thought about myself. Thanks!

File: charge-holy-weapon.lic [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+
+class ChargeHolyWeapon
+  include DRC
+  include DRCT
+
+  def initialize
+    exit unless DRStats.paladin?
+
+    @settings = get_settings
```
[the remark this answers is not in the document]
Comment (jayRyan24) [reply charge-holy-weapon.lic:14]: I thought I might use it later, but makes sense to not control it instance wide.

**Comment by jayRyan24** [comment]
Just got an icon. I will add to this.

**Comment by jayRyan24** [comment]
Added icon support, and changed up the layout of the hash used for the data.

File: charge-holy-weapon.lic [hunk 2]
```diff
@@ -38,27 +47,74 @@ class ChargeHolyWeapon
...
       message "You're not ready to recharge yet, exiting!"
     end
 
     exit
   end
+
+  def icon_ritual(weapon, name, room = @altar_room, container)
+    walk_to(room)
+
+    bput('open my #{container}', 'You open')
```
Comment (rpherbig) [inline charge-holy-weapon.lic:85]: ❌ This won't do string interpolation unless it uses double quotes

File: charge-holy-weapon.lic [hunk 3]
```diff
@@ -38,27 +47,74 @@ class ChargeHolyWeapon
...
+    case bput("get my #{name}", 'You get', 'What were')
```
Comment (rpherbig) [inline charge-holy-weapon.lic:86]: 💡 If `container` was specified, is there any value to being more specific (e.g. `get my name from my container`)?

**Comment by jayRyan24** [comment]
@rpherbig I marked all the change requests as resolved, unsure what's going on with it still marking it as such. Unless you need to mark them as resolved? Thanks for the feedback and catches!

**Review by rpherbig — APPROVED** [review]

**Comment by rpherbig** [comment]
> I marked all the change requests as resolved, unsure what's going on with it still marking it as such. Unless you need to mark them as resolved?

Yep, I need to update my review. Done.

**Pull request merged**